### PR TITLE
fix(DB/Creature): fix gordunni/forest walker movement

### PR DESCRIPTION
**Pull request by jestermaniac**
## Changes Proposed:

since we dont be havin no solid source i just checked more or less erry guid for the radius to wander. i left some to 0 cuz imo they be guards and issa right that they stay in the place and not stroll around

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6959
- Closes https://github.com/chromiecraft/chromiecraft/issues/1210

## SOURCE:
none really, only thang we know is that they gotta be wandering

## Tests Performed:

- tested locally, all good.

## How to Test the Changes:

1. just go in the guids provided in the commit file and check mobs wandering

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1629752061588917431.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1629752061588917431');
+
+UPDATE `creature` SET `MovementType` = 1, `wander_distance` = 3  WHERE (`guid` IN (50173, 50175, 50176, 50177, 50179, 50180, 50194, 50195, 50196, 50198, 50203, 50205, 50206, 50215, 50216, 50217, 50218, 50222, 50227, 50230, 50235, 50244, 50250, 50251, 50252, 50253, 50254, 50291, 50297, 50307, 50313, 50314, 50317, 50319));
```
Comment (steenburgh) [inline data/sql/updates/pending_db_world/rev_1629752061588917431.sql:3]: It looks like creatures also have a MovementType in their templates - did you consider updating the movement type in `creature_template` as well?

https://www.azerothcore.org/wiki/creature#movementtype
https://www.azerothcore.org/wiki/creature_template#creature_template-MovementType

File: data/sql/updates/pending_db_world/rev_1629752061588917431.sql [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+UPDATE `creature` SET `MovementType` = 1, `wander_distance` = 15 WHERE (`guid` IN (50800, 50268, 50269, 50270, 50294));
+UPDATE `creature` SET `MovementType` = 1, `wander_distance` = 6  WHERE (`guid` IN (50314, 91739, 91738));
+
```
Comment (Winfidonarleyan) [inline data/sql/updates/pending_db_world/rev_1629752061588917431.sql:6]:
```suggestion
```

**Comment by Winfidonarleyan** [comment]
Del new lines maybe?

**Comment by jestermaniac** [comment]
saw plenty of sql here with newlines. whats wrong wit em? i think they be keepin readability of code 10 toes down imo

**Comment by steenburgh** [comment]
Oh wow I didn't know we had [SQL Standards](https://www.azerothcore.org/wiki/sql-standards)

**Comment by steenburgh** [comment]
> Make sure your queries are precise so you avoid changing something you do not want to edit. Rule of thumb, always include all primary keys in your WHERE clause.

Is GUID a primary key in the tables you're editing?

**Review by jestermaniac — COMMENTED** [review]
> 
> 
> > Make sure your queries are precise so you avoid changing something you do not want to edit. Rule of thumb, always include all primary keys in your WHERE clause.
> 
> Is GUID a primary key in the tables you're editing?

y one and only

**Comment by FrancescoBorzi** [comment]
what's the status of this?

**Comment by steenburgh** [comment]
As far as I know, this is waiting to be tested, and waiting to be approved by someone with write access

**Review by FrancescoBorzi — APPROVED** [review]

**Pull request merged**